### ISSUE TITLE
OpenFOAM installation problem

### ISSUE THREAD

**Issue by sololo312** [ladybug-tools/butterfly#221]
Hi!
    I have tried and excluded from all the problems already but it still doesn't work.Is it possible that because I am in China,so I can not connect the host?(But I also tired to use VPN but still not working)
    1.First pic is the PowerShell while installing OpenFOAM
    2.Second pic is when I opened the Docker after installation
    Thx for helping me out!

![default](https://cloud.githubusercontent.com/assets/18479232/25070756/c9f42802-22d9-11e7-8eb6-c3921bc5669b.PNG)

![1](https://cloud.githubusercontent.com/assets/18479232/25070757/c9f86cfa-22d9-11e7-90d5-c12007052c58.PNG)

**Comment by MikkiSeidenschnur** [comment]
@sololo312 I think that you might be right - something went wrong when trying to update the boot2docker it seems. Don't know how that is fixed though.

**Comment by mostaphaRoudsari** [comment]
Hi @sololo312, Did you try to load the image from the command line? You can see the directions [here](https://github.com/ladybug-tools/butterfly/wiki/Windows-10-Installation-Notes#installing-openfoam-on-a-pc-using-windows-10). Check number 2.

**Comment by sololo312** [comment]
Thx ,@mostaphaRoudsari, I will see what I can do.Will updated my result.

**Comment by sololo312** [comment]
Let me try Mostapha's way.Thx!@MikkiSeidenschnur

**Comment by sololo312** [comment]
Hello there.Things went well just fine this time and I have no idea why.I did do anything else and I only doing the normal procedure.Now my Butterfly is working!I think about the problem happened before, there are some possibilities:

1. (Most suspect)I was running proxy software(XX Net)so I could not download the updates successfully,When I closed it this time,Everything goes fine.
2.Closed virus software before installation might help(Because the installation processes in the system's sensitive area)

Anyway,I am gonna closed the issue now.Hope it can help someone else!
After all,Thx for your help and concern @MikkiSeidenschnur and @mostaphaRoudsari .
Best.
